components/raftstore/src/store/worker/compact.rs

```diff
@@ -378,17 +391,31 @@ where
                     error!("execute compact range failed"; "cf" => cf, "err" => %e);
                 }
             }
+            Task::CollectWholeRangeMVCCStats { callback } => {
+                match self.engine.get_whole_range_stats(CF_WRITE) {
```

Comment:
When `write_cf` contains numerous SST files, the compact worker may become overloaded, potentially delaying `mvcc_stats `updates during `get_whole_range_stats`.

Maybe we should consider concurrent access and updates to mitigate this bottleneck ?